style.css

```diff
@@ -147,54 +156,266 @@
+  border-radius: 10px;
+  padding: 10px;
+}
+
+.work {
+  background-color: #e5e5e5;
+  width: 100%;
 }
 
-.tonic-img {
-  margin-left: 27px;
+.work-img {
   display: flex;
   flex-direction: column;
   justify-content: center;
-  height: 370px;
-  width: 370px;
+  margin: 10px 10px;
 }
 
-.tonic-text {
+.project-title {
+  margin: 5px 10px;
+  display: flex;
   flex-direction: column;
+  align-items: flex-start;
+  font-size: 42px;
+  color: rgb(0, 10, 47);
+  font-family: "Poppins", sans-serif;
+  font-weight: bold;
+  margin-top: 12px;
+  margin-left: 15;
+}
+
+.some-list {
+  margin: 5px 10px;
+  margin-left: 15px;
+}
+
+.work-info {
+  display: flex;
+  flex-direction: row;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+  padding: 15px;
+}
+
+.lang-list {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  width: 82%;
+  margin-left: -5%;
+  padding: 4px 12px;
+}
+
+.Html {
+  margin: 12px 2px;
+  padding: 5px 12px;
+  font-size: 16px;
+  color: #6070ff;
+  background-color: #ebebff;
+  border: 2px solid #ebebff;
+  border-radius: 9px;
+  outline: none;
+}
+
+.Css {
+  margin: 12px 2px;
+  padding: 5px 10px;
+  font-size: 16px;
+  color: #6070ff;
+  background-color: #ebebff;
+  border: 2px solid #ebebff;
+  border-radius: 9px;
+  outline: none;
+}
+
+.js {
+  margin: 12px 2px;
+  padding: 5px 10px;
+  font-size: 20px;
+  color: #6070ff;
+  background-color: #ebebff;
+  border: 2px solid #ebebff;
+  border-radius: 9px;
+  outline: none;
+}
+
+.button {
+  margin: 1.3rem;
+  padding: 0.7rem 1.5rem;
+  font-size: 20px;
+  color: #6070ff;
+  background-color: #fff;
+  border: 2px solid blue;
+  border-radius: 7px;
+  outline: none;
+  cursor: pointer;
+}
+
+.button:hover {
+  opacity: 0.7;
+}
+
+.button:active {
+  opacity: 0.3;
+}
+
+#project_1 {
+  margin: 10px 10px;
+}
+
+#project_2 {
+  margin: 10px 10px;
+}
+
+#project_3 {
+  margin: 10px 10px;
+}
+
+#project_4 {
+  margin: 10px 10px;
+}
+
+.aboutme_container {
+  background-color: #fff;
+  margin: 10px;
+  padding: 10px;
+  border: 3px solid #fff;
+  border-top-right-radius: 25% 50px;
+}
+
+.about_me {
+  display: flex;
+  margin: 70px 10px 40px 40px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: rgb(0, 9, 41);
+  text-shadow: steelblue;
+}
+
+.intro_2 {
   display: flex;
   justify-content: center;
+  margin: 10px 10px 40px 40px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.social-text_2 {
+  margin-left: 40px;
+  color: #6070ff;
+  font-size: 24px;
+}
+
+.social-icon_2 {
+  margin-top: -4px;
+  padding: 2px 5px;
+  margin-left: 15px;
+  display: flex;
+  justify-content: start;
   font-size: 30px;
-  margin-top: -30;
-  font-weight: bolder;
-  margin-left: -110px;
```

Comment:
- [ ] Kindly fix the color of the `icons` and use the correct `icon` for `medium link`. You can download them from [Figma Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1&t=TwQwsTNCrgvEHLHB-0) and use them to match the professional design.
![Screenshot from 2023-01-13 22-02-59](https://user-images.githubusercontent.com/60083437/212365240-b063c05f-d2a8-49a7-acd6-a8ec5ef697ca.png)